server/xpub-model/entities/auditLog/index.js

```diff
@@ -12,20 +12,18 @@ class AuditLog extends BaseModel {
         userId: { type: 'uuid' }, // Currently not used
         action: {
           type: 'string',
-          enum: ['CREATED', 'UPDATED', 'DELETED'],
+          enum: ['CREATED', 'UPDATED', 'DELETED', 'RECORD'],
```

Comment:
After a discussion here ... we're thinking that:
 -  'CREATED', 'UPDATED', 'DELETED' applies to database operations
 - Any other action should be explicit i.e. `MECA_RESULT`, `S3_UPLOADED`, `SFTP_UPLOADED` etc